### PR TITLE
[EventGrid] Fix lint errors

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py [hunk 1]
```diff
@@ -21,13 +23,17 @@ def generate_shared_access_signature(topic_hostname, shared_access_key, expirati
             Similar to <YOUR-TOPIC-NAME>.<YOUR-REGION-NAME>-1.eventgrid.azure.net
         :param str shared_access_key: The shared access key to be used for generating the token
         :param datetime.datetime expiration_date_utc: The expiration datetime in UTC for the signature.
-        :param str api_version: The API Version to include in the signature. If not provided, the default API version will be used.
+        :param str api_version: The API Version to include in the signature.
+        If not provided, the default API version will be used.
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py:27]: Need a space to keep it part of the param doc

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py [hunk 2]
```diff
@@ -75,7 +71,7 @@ class CloudEvent(EventMixin):   #pylint:disable=too-many-instance-attributes
      unique for each distinct event.
     :type id: Optional[str]
     """
-    def __init__(self, source, type, **kwargs):
+    def __init__(self, source, type, **kwargs): # pylint: disable=redefined-builtin
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py:74]: Why is that?

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py [hunk 3]
```diff
@@ -154,7 +150,8 @@ class EventGridEvent(InternalEventGridEvent, EventMixin):
     :param id: Optional. An identifier for the event. The combination of id and source must be
      unique for each distinct event.
     :type id: Optional[str]
-    :param event_time: Optional.The time (in UTC) of the event. If not provided, it will be the time (in UTC) the event was generated.
+    :param event_time: Optional.The time (in UTC) of the event. If not provided,
+    it will be the time (in UTC) the event was generated.
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py:154]: need a space here

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py [hunk 4]
```diff
@@ -25,18 +25,17 @@
...
-
 
 class EventGridPublisherClient(object):
     """EventGrid Python Publisher Client.
 
     :param str topic_hostname: The topic endpoint to send the events to.
-    :param credential: The credential object used for authentication which implements SAS key authentication or SAS token authentication.
-    :type credential: Union[~azure.core.credentials.AzureKeyCredential, azure.eventgrid.EventGridSharedAccessSignatureCredential]
+    :param credential: The credential object used for authentication which
+    implements SAS key authentication or SAS token authentication.
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py:34]: need a space here

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py [hunk 5]
```diff
@@ -25,18 +25,17 @@
...
+    :type credential: Union[
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py:35]: Not the docstring syntax:
```
:type credential: ~azure.core.credentials.AzureKeyCredential or ~azure.eventgrid.EventGridSharedAccessSignatureCredential
```

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py [hunk 6]
```diff
@@ -54,7 +53,8 @@ def send(self, events, **kwargs):
         :param events: A list or an instance of CloudEvent/EventGridEvent/CustomEvent to be sent.
         :type events: SendType
         :keyword str content_type: The type of content to be used to send the events.
-        Has default value "application/json; charset=utf-8" for EventGridEvents, with "cloudevents-batch+json" for CloudEvents
+        Has default value "application/json; charset=utf-8" for EventGridEvents,
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py:56]: space

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py [hunk 7]
```diff
@@ -3,16 +3,12 @@
...
 import uuid
 import json
 import six
-from ._generated import models
-from ._generated.models import StorageBlobCreatedEventData, \
-    EventGridEvent as InternalEventGridEvent, \
-    CloudEvent as InternalCloudEvent
+from msrest.serialization import UTC
+from ._generated.models import EventGridEvent as InternalEventGridEvent, CloudEvent as InternalCloudEvent
 from ._shared.mixins import DictMixin
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py:12]: Are we still using DictMixin at all?

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py (hunk 2 above)
Comment (rakshith91) [reply sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py:74]: `type` - unfortunately cant rename this either given cloud event spec

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_publisher_client_async.py [hunk 8]
```diff
@@ -36,8 +31,12 @@ class EventGridPublisherClient(object):
     """Asynchronous EventGrid Python Publisher Client.
 
     :param str topic_hostname: The topic endpoint to send the events to.
-    :param credential: The credential object used for authentication which implements SAS key authentication or SAS token authentication.
-    :type credential: Union[~azure.core.credentials.AzureKeyCredential, azure.eventgrid.EventGridSharedAccessSignatureCredential]
+    :param credential: The credential object used for authentication which implements
+    SAS key authentication or SAS token authentication.
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_publisher_client_async.py:35]: space

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_publisher_client_async.py [hunk 9]
```diff
@@ -36,8 +31,12 @@ class EventGridPublisherClient(object):
...
+    :type credential: Union[
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_publisher_client_async.py:36]: not docstring syntax

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_publisher_client_async.py [hunk 10]
```diff
@@ -55,7 +54,8 @@ async def send(self, events, **kwargs):
         :param  events: A list or an instance of CloudEvent/EventGridEvent/CustomEvent to be sent.
         :type events: SendType
         :keyword str content_type: The type of content to be used to send the events.
-        Has default value "application/json; charset=utf-8" for EventGridEvents, with "cloudevents-batch+json" for CloudEvents
+        Has default value "application/json; charset=utf-8" for EventGridEvents,
```
Comment (lmazuel) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_publisher_client_async.py:57]: space

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py (hunk 7 above)
Comment (rakshith91) [reply sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py:12]: we are using for the CustomEvent model

**Review by lmazuel — APPROVED** [review]

**Pull request merged**